sycl/include/sycl/ext/intel/esimd/math.hpp

```diff
@@ -96,8 +96,13 @@ namespace detail {
 template <typename T0, typename T1, int SZ>
```

Comment:
Generally speaking, this code is hard to read because 'T0' is used for result' and 'T1' for 'src0'.
It would be easier to have 'TRes' and 'TArg'